### PR TITLE
Tixati Torrent with OpenVPN

### PR DESCRIPTION
**Pull request by h1f0x**
Hi all
Created a docker image for Tixati (yes, working fine :D) with OpenVPN integration!

https://github.com/h1f0x/tixati-openvpn
https://github.com/h1f0x/tixati-openvpn-plexguide

I will put it into the forums for feedback once the upgrade is over.. i'm not able to atm :)

Cheers
h1f0x

### CODE REVIEW

**Comment by Admin9705** [comment]
thanks for contributing!

**Comment by Admin9705** [comment]
cannot add until home :(

**Comment by h1f0x** [comment]
no porblem at all mate :)

File: apps/tixati-openvpn [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        extport: '5822'
+        intport2: '5902'
+        extport2: '5922'
+        intport3: '3129'
+        extport3: '3149'
+        image: 'h1f0x/tixati-openvpn'
+
+    # CORE (MANDATORY) ############################################################
+    - name: 'Including cron job'
+      include_tasks: '/opt/plexguide/containers/_core.yml'
```
Comment (robertbaker) [inline apps/tixati-openvpn]: opt/communityapps/apps/_core.yml

File: apps/tixati-openvpn [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    - name: 'Including plugins'
+      include_tasks: '/opt/plexguide/containers/_plugins.yml'
```
Comment (robertbaker) [inline apps/tixati-openvpn]: opt/communityapps/apps/_plugins.yml

File: apps/tixati-openvpn [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    - name: 'Including folders'
+      include_tasks: '/opt/plexguide/containers/_downloaders.yml'
```
Comment (robertbaker) [inline apps/tixati-openvpn]: /opt/communityapps/apps/_downloadersyml

**Comment by robertbaker** [comment]
@h1f0x you're better off  making a PR for this against v8.5 branch but you still need to make fixes before we can merge this

**Comment by Admin9705** [comment]
ya set to the wrong branch or edition.

**Comment by Admin9705** [comment]
closing this for now :D

**Pull request closed without merging**